### PR TITLE
feat: implements toggle component

### PR DESCRIPTION
**Pull request by bladicito**
https://github.com/ost-cas-fee-adv-23-24/design-system-component-library-football-fanatics/issues/16

### CODE REVIEW

File: stories/Button.stories.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { IconOptions } from './Icon.stories';
 
 // More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
 const meta: Meta<typeof Button> = {
-  title: 'Components/Button',
+  title: 'Components/Components/Button',
```
Comment (lehmi11) [inline stories/Button.stories.ts:14]: Why do create another Components folder for the storybook?

File: components/toggle/css.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  iconContainer: ['w-4', 'h-4', 'mr-2'],
+};
+
+export const likeStates = {
+  hover: {
+    topContainer: ['hover:bg-pink-50', 'hover:text-pink-600'],
+    iconContainer: ['text-pink-500'],
+  },
+  active: {
+    topContainer: ['active:text-pink-900', 'text-pink-900'],
```
Comment (lehmi11) [inline components/toggle/css.ts:35]: is there a `active: `missing in front of text-pink-900?

**Review by lehmi11 — APPROVED** [review]
Some minor questions. otherwise nice job 👍

File: components/toggle/css.ts (hunk 2 above)
Comment (bladicito) [reply components/toggle/css.ts:35]: no. it is correct. the active state applies also to a like with likes already.

File: stories/Button.stories.ts (hunk 1 above)
Comment (bladicito) [reply stories/Button.stories.ts:14]: no special idea here. we can also keep them on the root.  I thought to make a separation between base components like typo, icons. for me no problem at all to revert this.

**Pull request merged**